src/chttpd/src/chttpd_misc.erl

```diff
@@ -141,6 +144,38 @@ all_dbs_callback({error, Reason}, #vacc{resp=Resp0}=Acc) ->
     {ok, Resp1} = chttpd:send_delayed_error(Resp0, Reason),
     {ok, Acc#vacc{resp=Resp1}}.
 
+handle_dbs_info_req(#httpd{method='POST'}=Req) ->
+    chttpd:validate_ctype(Req, "application/json"),
+    Props = couch_httpd:json_body_obj(Req),
```

Comment:
I think it would be better if you would use functions which are defined in the current application. The `chttpd:json_body_obj/1` provides the same functionality.

Reply:
changed to chttpd:json_body_obj/1